data/styles/banner.css

```
@@ -48,41 +48,38 @@
 }
 
 .banner button {
-    background-color: alpha(@banner_fg_color, 0.6);
-    background-image: none;
-    border-color: alpha(@banner_fg_color, 0.7);
-    box-shadow: none;
+    background-color: @banner_fg_color;
```

I would also get rid of the background-image tbh
```suggestion
    background-color: @banner_fg_color;
    background-image: none;
```